module/project/templateTasks/helper.js

```diff
@@ -6,888 +6,799 @@
  */
 
 /**
-    * ProjectTemplateTasksHelper
-    * @class
-*/
+ * ProjectTemplateTasksHelper
+ * @class
+ */
 
 // Dependencies
-const learningResourcesHelper = require(MODULES_BASE_PATH + "/learningResources/helper");
-const projectTemplateTaskQueries = require(DB_QUERY_BASE_PATH + "/projectTemplateTask");
-const projectTemplateQueries = require(DB_QUERY_BASE_PATH + "/projectTemplates");
-const solutionsQueries = require(DB_QUERY_BASE_PATH + "/solutions");
+const learningResourcesHelper = require(MODULES_BASE_PATH + '/learningResources/helper')
```

Comment:
@MallanagoudaB What is the change in this file or is it just formatting?

Reply:
i didnt added any thing there might be formatting